docs/cloud/dev/spryker-cloud-commerce-os/multi-store-setups.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Multi-store setups
+description: Types of multi-store setups in Spryker Cloud Commerce OS
+template: concept-topic-template
+---
+
+This document describes the multi-store setups supported by Spryker Cloud Commerce OS.
+
+Currently, the following setups are available:
+
+* Separated: a codebase with dedicated databases per store.
+* Shared: a codebase with all the stores sharing a database.
+
+
+## Shared setup
+
+With the shared setup, all the stores share a single database and codebase.
```

Comment:
Share single database per region as it's possible to have EU and US regions, at least for local development